src/mame/misc/micom_mahjong.cpp

```diff
@@ -93,7 +93,7 @@ void mmahjong_state::vram_w(offs_t offset, u8 data)
 
 void mmahjong_state::input_w(u8 data)
 {
-	// d0-d3: input mux
+	// d0-d2: input mux
```

Comment:
It’s not a multiplexer, though – it’s just an active-low key matrix.

Reply:
Just a second